instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/internal/LoggingEventMapper.java

```diff
@@ -104,6 +108,17 @@ private void mapLoggingEvent(LogRecordBuilder builder, ILoggingEvent loggingEven
       attributes.put(SemanticAttributes.THREAD_ID, currentThread.getId());
     }
 
+    if (captureCodeAttributes) {
+      StackTraceElement[] callerData = loggingEvent.getCallerData();
+      if (callerData != null && callerData.length > 0) {
+        StackTraceElement firstStackElement = callerData[0];
+        attributes.put(SemanticAttributes.CODE_FILEPATH, firstStackElement.getFileName());
```

Comment:
`StackTraceElement#getFileName()` might return null, let's null-check it before passing it to `attributes`

Reply:
Done, I did not know that null attributes have not to bet set